Markup kadai 08

blue2wo opened the pull request:
練習課題8「Amazon登録画面」1st commit
35f6911 HTML CSS

piasoratech commented on markup_kadai_08/markup_kadai_08.html:
```diff
@@ -0,0 +1,76 @@
+    <link rel="stylesheet" href="styleE8.css">
+</head>
+<body>
+    <!-- page container   -->
```
`main`タグを使用しましょう

piasoratech commented on markup_kadai_08/markup_kadai_08.html:
```diff
@@ -0,0 +1,76 @@
+    <div id="base">
+        
+        <!--  amazon log      -->
+        <div id="amazonlogo"></div>
```
ロゴ画像などを入れる場合は`img`タグを使用するほうが一般的です。

piasoratech commented on markup_kadai_08/markup_kadai_08.html:
```diff
@@ -0,0 +1,76 @@
+        
+        <!-- form container       -->
+        <div id="outer">
+            <div id="inner">
```
ここの`div`タグは使用しなくても、`id="outer"`のほうのCSSを変更すれば実現できるのでやってみましょう。

piasoratech commented on markup_kadai_08/markup_kadai_08.html:
```diff
@@ -0,0 +1,76 @@
+                        <input type="submit" value="Amazonアカウントを作成" id="submitbutton">
+                        <div class="filler"></div>
+                        <p>ログインすることにより、当社の<a href="https://www.amazon.co.jp/gp/help/customer/display.html?nodeId=201909010">利⽤規約</a>および<a href="https://www.amazon.co.jp/gp/help/customer/display.html?nodeId=201909010">プライバシー規約</a>に同意したとみなされます。</p>
+                        <hr class="fade_rule">
```
この線はどのような意図で追加しましたか？

piasoratech commented on markup_kadai_08/markup_kadai_08.html:
```diff
@@ -0,0 +1,76 @@
+        <div class="container">
+            <div class="item" class="helplinks"><a href="https://www.amazon.co.jp/gp/help/customer/display.html?nodeId=201909000">利⽤規約</a></div>
+            <div class="item" class="helplinks"><a href="https://www.amazon.co.jp/gp/help/customer/display.html?nodeId=201909010">プライバシー規約</a></div>
+            <div class="item" class="helplinks"><a href="https://www.amazon.co.jp/gp/help/customer/display.html/ref=jp_surl_help?nodeId=508510">ヘルプ</a></div>
```
`class`の複数指定の仕方はこのやり方ではないので、修正しましょう

piasoratech commented on markup_kadai_08/styleE8.css:
```diff
@@ -0,0 +1,139 @@
+body {
+/*    line-height: 25px;*/
```
不要なコードは含めないようにしましょう！

piasoratech commented on markup_kadai_08/styleE8.css:
```diff
@@ -0,0 +1,139 @@
+
+h1 {
+    margin: 0 0 20px 0;
+/*    padding: 0;*/
```
ここも同様です

piasoratech commented on markup_kadai_08/styleE8.css:
```diff
@@ -0,0 +1,139 @@
+    font-size: 0.9em;
+    padding: 10px 0 10px 5px
+*/
+}
```
適用されるデザインがない場合は項目自体も削除しましょう

piasoratech commented on markup_kadai_08/styleE8.css:
```diff
@@ -0,0 +1,139 @@
+#outer {
+    border: 1px #ddd solid;
+    border-radius: 4px;
+/*    box-shadow: inset 0 1.3px 1px #ddd;*/
```
ここも同様です

piasoratech commented on markup_kadai_08/styleE8.css:
```diff
@@ -0,0 +1,139 @@
+ 
+.inputform {
+    width: 98%;
+/*    line-height: 35px;*/
```
ここも同様です

blue2wo commented:
6e27dbe
- removed unnecessary code and other kadai files. Only kadai 08 files remain.
- used `<main>` tags
- changed `<div id="amazonlogo"></div>` to `<img>` tag
- removed `<div id="inner">` and changed CSS of `<div` id="outer">`
- removed `<hr class="fade_rule">` 
- removed multiple classes `<div class="item" class="helplinks">`

blue2wo commented:
練習課題8「Amazon登録画面」3rd push - feedback changes using https://validator.w3.org/#validate_by_input

97020c6
- added `alt` tag for images
- moved `<input>` from `<ul>` child to inside `<li>`
- removed duplicated `id`

fjtawork commented on markup_kadai_08/markup_kadai_08.html:
```diff
@@ -0,0 +1,79 @@
+                </div>
+            </form>
+        </div>
+        <div class="filler"></div>
```
スペースを空けるために中身の無いdivタグを使うのは、あんまり良くないですね。
margin-bottomとかを使って、空白を作りましょう！
